openstudiocore/src/model/ElectricLoadCenterDistribution.cpp

```diff
@@ -479,8 +481,14 @@ namespace detail {
 
 
   // Storage Converter Object Name
-  //bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter);
-  //void ElectricLoadCenterDistribution_Impl::resetStorageConverter();
+  bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter) {
```

Comment:
Why do you say this?

I think it kinda makes sense in this way, you can easily see the ELCD with his inverter, transformer an storage devices. It'd be more abstract and hard to comprehend the other way around. Also this allows you to see the bussType of the elcd and quickly see if that makes sense considering your objects.

Speaking about this, we've scratch the subject before, but maybe we should switch the bussType automatically based on whether you set/reset inverters and converters. I don't really need that (my validityCheck method is enough for me), and I have little time to do this now, but this is food for thoughts.



Envoyé de mon iPhone

> Le 30 juil. 2016 à 01:03, Dan Macumber <notifications@github.com> a écrit :
> 
> In openstudiocore/src/model/ElectricLoadCenterDistribution.cpp:
> 
> > @@ -479,8 +481,14 @@ namespace detail {
> >  
> >  
> >    // Storage Converter Object Name
> > -  //bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter);
> > -  //void ElectricLoadCenterDistribution_Impl::resetStorageConverter();
> > +  bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter) {
> Really E+ should have made the storage converter point to the electric load center
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>